scripts/tests/local.py

```diff
@@ -778,6 +798,19 @@ def gen_coverage(flat):
     type=click.Choice(list(__RUNNERS__.keys()), case_sensitive=False),
     help="Determines the verbosity of script output",
 )
+@click.option(
+    "--override-binary-path",
+    default=None,
+    nargs=2,
+    multiple=True,
+    help="Defines an override binary path for a given app. Can be used multiple times. E.g. --override-binary-path ALL_CLUSTERS_APP out/some/path/chip-all-clusters-app"
+)
+@click.option(
+    "--app-filter",
+    default="*",
```

Comment:
If we use "*" does this imply we support globbing? Should we mark it as string and handle None/empty instead?